crates/forge/src/lib.rs

```diff
@@ -441,6 +443,8 @@ async fn run_tests_from_crate(
         results.push(result);
     }
 
+    rec.close();
```

Comment:
Why do we close instead of dropping?

Reply:
https://docs.rs/tokio/latest/tokio/sync/mpsc/struct.Receiver.html - I want to close a channel. Not remove single sender